File: src/main/java/model/equipment/armor/Gloves.java
package model.equipment.armor;

import model.support.allEnums.ArmorMaterial;
import model.support.allEnums.ArmorType;
import model.support.allEnums.EquipmentType;

public class Gloves extends Armor {
    public Gloves(ArmorMaterial material, double weight, int quality) {
        super(EquipmentType.ARMOR, ArmorType.GLOVES, material, weight, quality);
    }
}

File: src/main/java/model/equipment/armor/Gloves.kt
package model.equipment.armor

import model.support.enums.ArmorMaterial
import model.support.enums.ArmorType.GLOVES
import model.support.enums.BodyPart
import model.support.enums.BodyPart.HANDS
import model.support.enums.EquipmentType.ARMOR

class Gloves(material: ArmorMaterial, weight: Double, quality: Int) : Armor(ARMOR, GLOVES, material, HANDS, weight, quality)